src/main/java/org/folio/services/budget/BudgetService.java

```diff
@@ -102,35 +107,42 @@ public Future<Void> deleteBudget(String id, RequestContext requestContext) {
   }
 
   private void validateBudget(SharedBudget budget) {
+    log.debug("validateBudget:: Validation budget");
     List<Error> errors = new ArrayList<>();
 
     errors.addAll(checkRemainingEncumbrance(budget));
     errors.addAll(checkRemainingExpenditure(budget));
 
     if (!errors.isEmpty()) {
+      log.error("validateBudget:: Error found during validation");
```

Comment:
similar as in previous places, for error logs better to add the main identifier also, in this case budgetId to quickly search by this budgetId

Reply:
Thanks, added